models/lido/liquidity/scroll/lido_liquidity_scroll_zebra_pools.sql

```diff
@@ -0,0 +1,287 @@
+{{ config(
+    schema='lido_liquidity_scroll',
+    alias = 'zebra_pools',
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['pool', 'time'],
+    post_hook='{{ expose_spells(\'["scroll"]\',
+                                "project",
+                                "lido_liquidity",
+                                \'["pipistrella"]\') }}'
```

Comment:
Same as above.

Reply:
Thanks a lot for your review, @Hosuke 
I have updated code according to your comments